src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -1335,13 +1496,11 @@ public void setSoLinger(boolean on, int linger) throws SocketException {
         }
 
         public void setSoTimeout(int timeout) throws SocketException {
-            if (logger.isLoggable(Level.FINER))
-                logger.finer(toString() + " Ignoring setSoTimeout");
+            tdsChannel.tcpSocket.setSoTimeout(timeout);
         }
 
         public void setTcpNoDelay(boolean on) throws SocketException {
-            if (logger.isLoggable(Level.FINER))
-                logger.finer(toString() + " Ignoring setTcpNoDelay");
+            tdsChannel.tcpSocket.setTcpNoDelay(on);
```

Comment:
Similar to the above... I can't think of a good reason we were ignoring this when encrypted but applying it when not.